src/api/v2/quickPickWizard/SkipIfOneQuickPickStep.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ResourceModelBase } from '../v2AzureResourcesApi';
+import { QuickPickWizardContext } from './QuickPickWizardContext';
+import { RecursiveQuickPickStep } from './RecursiveQuickPickStep';
+
+export class SkipIfOneQuickPickStep<TModel extends ResourceModelBase> extends RecursiveQuickPickStep<TModel> {
+    public override async prompt(wizardContext: QuickPickWizardContext<TModel>): Promise<void> {
+        const picks = await this.getPicks(wizardContext);
+
+        if (picks.length === 1) {

```

Comment:
I mentioned some other criterion for skipping in a prior comment

Reply:
I think it needs to be determined by the consumer and vary depending on the scenario.